src/app/shared/services/http.service.ts

```diff
@@ -85,6 +85,10 @@ export class HttpService {
       .pipe(catchError(this.handleError()));
   }
 
+  getReportsInProgressThresholdTime() {
+    return this.http.get<any>('api/testtool/in-progress/threshold-time').pipe(catchError(this.handleError()));
```

Comment:
It hurts so much to see `any` here, are you sure you dont know the type?
Maybe something you and @MatthijsSmets can consider; using `unknown` instead of any so at least some form of type checking can be applied instead of having `any` which disables type checking for those objects

Reply:
I changed it to number